index.js

```diff
@@ -24,6 +24,7 @@ var $ = document.querySelector.bind(document);
 
 // Globals
 var logo, turtle;
+var examples = 'examples.txt';
```

Comment:
Is the idea here that a later script will override this, before the load handler runs?